packages/jest-preset-default/package.json

```diff
@@ -31,10 +31,10 @@
 	],
 	"main": "index.js",
 	"dependencies": {
+		"@wojtekmaj/enzyme-adapter-react-17": "^0.3.1",
```

Comment:
So it looks like Airbnb no longer maintains enzyme ...

Reply:
It's community maintained, but the PR for React 17 support is still WIP: https://github.com/enzymejs/enzyme/pull/2430

Reply:
Well, apparently the unofficial adapter we're using has an ongoing issue with `unmount` not working as expected, so that's why the `<ScrollLock>` test is failing: https://github.com/wojtekmaj/enzyme-adapter-react-17/issues/12. I'll check for progress there. Official PR is still WIP.

Reply:
FYI, this test should now work as the bugifx was just merged & released. Please update and check!